### ISSUE TITLE
authentification always fail

### ISSUE THREAD

**Issue by chris93111** [rancher-plugins/kontainer-engine-driver-oke#20]
Hi

i try to deploy oke with rancher but i have always error 401 in authentification

i have try with many user 

my credential work correctly with sdk

Service error:NotAuthenticated. The required information to complete authentication was not provided or was incorrect.. http status code: 401. Opc request id: XXXXXXXXXX

**Comment by chris93111** [comment]
Hi it's work with 14.0 , i think region parameter is missing in 1.4.2

rancher 2.5

**Comment by jlamillan** [comment]
Thanks for the report, we'll check it out￼

**Comment by jlamillan** [comment]
@jrosinsk ￼

**Comment by jlamillan** [comment]
@chris93111 - I have not been able to reproduce this issue on Rancher `v2.5.1` (using the `v1.4.2` OKE driver which is installed by default).

Is there anything else specific to your environment that I should be trying?  For example, is your OCI API password protected? Are you using a different OKE driver version? etc.

**Comment by chris93111** [comment]
Hi i don't have the parameter region with 2.5.2 and oci 1.4.2 , work with 2.4.8 because parameter "region" is available
![image](https://user-images.githubusercontent.com/27526373/99074494-22069900-25b8-11eb-8563-ab695fe30312.png)

**Comment by jlamillan** [comment]
I believe its choosing `us-phoenix-1`, which should be fine for authentication purposes.

Is the `admin` in the User OCID field typo? Note that it is expecting a user _OCID_ i.e. `ocid1.user.oc1..aaaaaaaa....`

**Comment by chris93111** [comment]
yes my ocid is ok because i can use with sdk of oci and api with no problem

but my region is in frankfurt, so maybe the region phoenix is not same authentification and i can't change this parameter with 2.5.2

**Comment by jlamillan** [comment]
Ok. Using the console, you should be able to check whether you're "subscribed" to the Phoenix region￼

**Comment by jlamillan** [comment]
What you're describing does sound like a bug. But we have a bit of a ￼"chicken and egg" problem in that we need to authenticate to retrieve the region list.￼

The region list that was there before was static and often out of date since we're adding new regions so frequently.￼

**Comment by chris93111** [comment]
I don't have subscription with phoenix region
Ok i inderstand region is recover dynamically but with 2.4.8 the region parameter is set by the user , maybe you can let the user overload region in 2.5.1 and with empty param use dynamic search region

**Comment by jlamillan** [comment]
@jrosinsk

**Comment by jlamillan** [comment]
While we are sorting this issue out, you should be able to work around your authentication issue by subscribing to the "US West (Phoenix)" region:

OCI Console=> Administration => Region Management=> US West (Phoenix) => Subscribe

**Comment by jlamillan** [comment]
[Upstream PR](https://github.com/rancher/ui/pull/4324) to add the list of regions back to the authentication pane. 

![OKE Cluster](https://user-images.githubusercontent.com/13613687/100946263-d7da4e80-34b7-11eb-9ee0-6db751e33bc9.png)
